src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -102,6 +120,56 @@ class PaymentInformation extends React.Component {
     }
   }
 
+  handleDirectDepositUpdateSubmit(data) {
+    this.props.savePaymentInformation(data);
+    recordEvent({
+      event: 'profile-transaction',
+      'profile-section': 'direct-depost-information',
+    });
+  }
+
+  handleEditClick(e) {
```

Comment:
If you do `handleEditClick = e => {`, you don't need to use `bind` when you call this down below. It automatically does the binding for you.

Reply:
thanks for pointing this out, Jeff. The same thing should apply to `handleDirectDepositUpdateSubmit` above on line 123, correct?

Reply:
Yep! I just missed that one.

Reply:
All fixed.  Thx for the method-signature trick, Jeff!